aries_cloudagent/anoncreds/issuer.py

```diff
@@ -1256,3 +1257,24 @@ async def revoke_credentials(
             curr=updated_list,
             failed=[str(rev_id) for rev_id in sorted(failed_crids)],
         )
+
+    async def update_revocation_list(
+        self,
+        tails_public_uri,
+        revoc_reg_def,
+        prev_list,
+        curr_list,
+        options: Optional[dict] = None,
+    ) -> RevListResult:
+        """Send a registry entry to the ledger."""
+
+        self._check_url(tails_public_uri)
+
+        anoncreds_registry = self._profile.inject(AnonCredsRegistry)
+        return await anoncreds_registry.update_revocation_list(
```

Comment:
In addition to updating the remote registry as we are here with the `anoncreds_registry.update_revocation_list`, we also need to update the local representation in our wallet.

Something like these lines that I pulled from `create_and_register_revocaiton_list`:

```python
        try:
            async with self._profile.session() as session:
                await session.handle.insert(
                    CATEGORY_REV_LIST,
                    rev_reg_def_id,
                    result.revocation_list_state.revocation_list.to_json(),
                )
        except AskarError as err:
            raise AnonCredsIssuerError("Error saving new revocation registry") from err
```